### PR TITLE
framework/araui: Fix typo and modify text rendering procedure

### PR DESCRIPTION
**Pull request by tyha1115**
- framework/araui: Fix typo in the AraUI.
UI_PIXEL_FORMAT_ARGA8888 -> UI_PIXEL_FORMAT_ARGB8888

- frameowkr/araui: Modify text rendering procedure.
Different from images, a bitmap of single text has only alpha information.
In order to provide rendering text feature with color, corresponding APIs are added.
(ui_renderer_set_fill_color())

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [7c59c8ad1d655cc465f014e8e8043c3fc0d7ba77] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [7c59c8ad1d655cc465f014e8e8043c3fc0d7ba77] - Code Rule Check (C++) OK.

File: framework/src/araui/widgets/ui_text_widget.c [hunk 1]
```diff
@@ -575,12 +581,40 @@ static void _ui_text_widget_render_func(ui_widget_t widget, uint32_t dt)
...
+				ui_renderer_set_fill_color(body->font_color);
+
+				v1 = (ui_vec3_t){
+					.x = -body->base.pivot_x,
+					.y = -body->base.pivot_y,
+					1.0f
+				};
+				v2 = (ui_vec3_t){
+					.x = -body->base.pivot_x,
+					.y = -body->base.pivot_y + body->base.local_rect.height,
```
Comment (gcjjyy) [inline framework/src/araui/widgets/ui_text_widget.c]: outw  / outh instead of local_rect

**Comment by seinfra** [comment]
Target : [27bc6f9f35fe5bf355c0e9316e3ac275e483ff4e] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [27bc6f9f35fe5bf355c0e9316e3ac275e483ff4e] - Code Rule Check (C++) OK.

**Pull request merged**